package.json

```diff
@@ -29,6 +29,7 @@
     "ngb:test": "ng test ng-bootstrap --code-coverage --source-map true --progress false --watch false",
     "ngb:tdd": "ng test ng-bootstrap --source-map false",
     "ngb:e2e": "ng e2e e2e-app",
+    "ngb:e2e-demo": "yarn demo:docs && ng e2e demo",
```

Comment:
please rename to `demo:e2e` / move lower to all other `demo:**` tasks for consistency with others